compiler/src/main/scala/org/scalajs/nscplugin/GenJSCode.scala

```diff
@@ -468,7 +447,7 @@ abstract class GenJSCode[G <: Global with Singleton](val global: G)
          */
```

Comment:
I think the comment above needs updating?

Reply:
Nice catch!